Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby '3.1.3'
+ruby '3.1.2'
```

Comment:
Kindly remove these changes made in Gemfile from the PR, it will create a conflict. You can do so with the following command:
```
$ git checkout model
$ git checkout dev -- Gemfile
$ git commit -m "Remove Gemfile file from PR"
$ git push origin model
```

Reply:
Hi  @amexabee 
these commands are not working and I think it's not a big problem 
when you pull you can just change your Ruby version